zetaclient/evm_client.go

```diff
@@ -92,7 +96,8 @@ type EVMChainClient struct {
 	params                     observertypes.ChainParams
 	ts                         *TelemetryServer
 
-	BlockCache *lru.Cache
+	BlockCache  *lru.Cache
```

Comment:
Do we need to persist these caches to the db? When you restart zetaclient you it will just have to query the block and header again.

Reply:
we don't need to persist is my understanding

Reply:
no need.